lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt

```diff
@@ -85,7 +137,27 @@ interface Account {
         maxFee: Felt,
         nonce: Felt,
         forFeeEstimate: Boolean,
-    ): DeployAccountTransactionPayload
+    ): DeployAccountTransactionV1Payload
+
+    /**
+     * Sign version 3 deploy account transaction.
+     *
+     * Sign a deploy account transaction that requires prefunding deployed address.
```

Comment:
```suggestion
     * Sign a deploy account transaction that requires prefunding said account.
```
Not sure deployed is the best wording